examples/gallery/symbols/text_symbols.py

```diff
@@ -10,10 +10,10 @@
 different characters) and *string* can be a letter or a text string
 (less than 256 characters). Optionally, you can append **+f**\ *font* to
 select a particular font [Default is :gmt-term:`FONT_ANNOT_PRIMARY`] as well as
-**+j**\ *justify* to change the justification [Default is CM]. Outline
-and fill color of the text symbols can be customized via the ``pen``
-and ``color`` parameters, respectively.
-
+**+j**\ *justify* to change the justification [Default is CM]. The outline
+color of the text symbols can be defined via the ``style`` parameter [Default
+is black], the fill color can be set with the ``color`` parameter, and the
```

Comment:
>  The outline color of the text symbols can be defined via the ``style`` parameter 

It's unclear how to change the outline color via the `style` parameter. Perhaps we should change the previous sentence to

> Optionally, you can append **+f**\ *font*,*outlinecolor* to select a particular font and outline color

Reply:
That's the best solution, done, thanks!